src/plugin/customParseFormat/index.js

```diff
@@ -118,6 +120,9 @@ function correctHours(time) {
 }
 
 function makeParser(format) {
+  if (format.match(/l/i)) {
+    format = format.replace(/(\[[^\]]+])|(LTS|LT|l+|L+)/g, (_, a, b) => a || locale.formats[b] || t(locale.formats[b.toUpperCase()]))
```

Comment:
just thinking should we export this logic from localizedFormat as well?

Reply:
it's now exported from there.